client/my-sites/site-settings/media-settings/index.jsx

```diff
@@ -74,13 +74,12 @@ const MediaSettings = ( {
 					disabled={ isRequestingSettings || isSavingSettings }
 					/>
 				<div className="media-settings__module-settings site-settings__child-settings">
-					<FormToggle
-						className="media-settings__carousel-module-settings-toggle is-compact"
+					<CompactFormToggle
 						checked={ fields.carousel_display_exif || false }
 						disabled={ isRequestingSettings || isSavingSettings || ! carouselActive }
 						onChange={ handleAutosavingToggle( 'carousel_display_exif' ) } >
 						{ translate( 'Show photo metadata in carousel, when available' ) }
```

Comment:
Thanks for clarifying @yoavf, but as per a discussion in Slack we've decided to ditch all dots from the field labels, regardless of the i18n bot suggestions.